EIPS/eip-draft_universal_router.md

```diff
@@ -0,0 +1,282 @@
+---
+eip: <to be assigned>
+title: Universal Router Contract
+description: Universal router contract designed for token allowance that eliminates all `approve` transactions in the future.
+author: Zergity <zergity@gmail.com>
```

Comment:
If you'd like to have both your email address and a GitHub username, you can do something like:

```suggestion
author: Zergity (@Zergity), Zergity <zergity@gmail.com>
```